### PR TITLE
fixing jsons in models

### PR DESCRIPTION
**Pull request by natashaarmbrust**

### CODE REVIEW

File: Podcast/Episode.swift [hunk 1]
```diff
@@ -13,26 +13,26 @@ class Episode: NSObject {
     
     var id: Int
     var title: String
-    var series: Series?
+    var seriesId: Int
```
Comment (dennisfedorko) [inline Podcast/Episode.swift]: seriesID should be a string

Comment (natashaarmbrust) [reply Podcast/Episode.swift]: Agreed all the ids should probably be

Comment (dennisfedorko) [reply Podcast/Episode.swift]: seriesID not seriesId

**Comment by dennisfedorko** [comment]
looks good, just fix that one thing and merge

**Pull request merged**